src/wallet/migrate.cpp

```diff
@@ -519,7 +582,7 @@ void BerkeleyRODatabase::Open()
                     uint32_t next_page = orec->page_number;
                     while (next_page != 0) {
                         SeekToPage(db_file, next_page, page_size);
-                        PageHeader opage_header;
+                        PageHeader opage_header(inner_meta.other_endian);
                         db_file >> opage_header;
                         OverflowPage opage(opage_header);
                         db_file >> opage;
```

Comment:
In commit "Implement handling of other endianness in BerkeleyRODatabase" 0b8eac6a78351a68c1b5a6126564493ab50031dc

I might be reading something wrong, or confusing something, but I don't quite understand why the record data itself does not have to get its endianness adjusted.

Reply:
The record data should just be arbitrary byte strings so they don't have an endianness.

Reply:
ACK, please resolve.